src/views/home/components/header/header.styles.ts

```diff
@@ -0,0 +1,65 @@
+import { createUseStyles } from "react-jss";
+
+import { Theme } from "src/styles/theme";
+
+const flexBox = { width: "100%", display: "flex" };
+const btn = (theme: Theme) => ({
+  display: "inline-flex",
+  alignItems: "center",
+  padding: [[theme.spacing(0.75), theme.spacing(1)]],
+  borderRadius: theme.spacing(1),
+  height: theme.spacing(4.5),
+  "& svg": {
+    width: 24,
+  },
+  "& p": {
+    margin: [[0, theme.spacing(1)]],
+    display: "none",
+    [theme.breakpoints.upSm]: {
+      display: "inherit",
+    },
+  },
+  "&:hover": {
+    backgroundColor: theme.palette.grey.light,
+  },
+});
+
+const useHeaderHomeStyles = createUseStyles((theme: Theme) => ({
+  header: {
+    ...flexBox,
+    justifyContent: "space-between",
+    fontWeight: theme.fontWeights.medium,
+    "& > *": {
+      width: "100%",
```

Comment:
Why is this required? I mean, the logo and the subtitle will naturally get all the space they need.

Reply:
it's need it to keep the logo in the real center with the Bridge text

Reply:
I may not get the point but I can't appreciate any difference:

![bridge-logo-centering](https://user-images.githubusercontent.com/15896177/156761701-fc143a10-89bd-45c1-a20d-568113eea29a.gif)

Reply:
![Peek 2022-03-04 13-44](https://user-images.githubusercontent.com/19190248/156765922-10070215-9ab9-4c6c-b4f6-120bd391828c.gif)

Reply:
Aha! it's a browser-specific issue!